assets/sass/pages/_talk.scss

```diff
@@ -65,7 +64,7 @@
           position: absolute;
           left: 3rem;                
           width: 0.25rem;
-          height: 20px;
+          height: 30px;
```

Comment:
This fixed the issue of the highlighted bar when you hover on the link, which looked smaller and off by a few pixels.